build/Makefile

```diff
@@ -102,11 +109,18 @@ build-images: build-controller-image build-agones-sdk-image
 build-sdks: build-sdk-cpp
 
 # Run all tests
-test: ensure-build-image test-go test-install-yaml
+test: $(ensure-build-image) test-go test-install-yaml
 
 # Run go tests
 test-go:
-	docker run --rm $(common_mounts) $(build_tag) go test -race $(agones_package)/...
+	docker run --rm $(common_mounts) $(build_tag) go test -race $(agones_package)/pkg/... \
+	$(agones_package)/sdks/...
+
+# Runs end-to-end tests on the current configured cluster
+test-e2e:
```

Comment:
We'll also need a `minikube-test-e2e` target, just because it has special mounting for .kube config and should run on host network. Should be able to lift an existing example, or I can write it if you want.

Reply:
let me see if I can get it working.